src/module.c

```diff
@@ -873,6 +888,7 @@ int TSDB_incrby(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
                 series->lastValue = 0;
             }
         }
+        currentUpdatedTime = max(currentUpdatedTime, series->lastTimestamp);
```

Comment:
@ashtul why did you move the max to the RESET block?

Reply:
Because it is relevant only if incrby has RESET flag.

Reply:
@ashtul what was the effect of that not only on RESET?

Reply:
@gkorland the max wouldn't do anything since (except if you are using MacOS docker :) currentUpdatedTime would always be greater than series->lastTimestamp.
With RESET, it was normalized to resetMilliSeconds